language-server/src/main/java/org/sonarlint/languageserver/SonarLintLanguageServer.java

```diff
@@ -531,8 +602,25 @@ public void didChangeWatchedFiles(DidChangeWatchedFilesParams params) {
     // No watched files
   }
 
-  public StandaloneSonarLintEngine getEngine() {
-    return engine;
+  // TODO this method never seems to get triggered...
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&pullRequest=191&issues=AWMHyHIHuCK54mAOyiXp&open=AWMHyHIHuCK54mAOyiXp)

[ISSUE_KEY]: AWMHyHIHuCK54mAOyiXp